src/cascadia/TerminalApp/CommandPalette.cpp

```diff
@@ -161,13 +164,25 @@ namespace winrt::TerminalApp::implementation
         if (key == VirtualKey::Up)
         {
             // Action Mode: Move focus to the next item in the list.
-            SelectNextItem(false);
+            SelectNextItem(false, false);
             e.Handled(true);
         }
         else if (key == VirtualKey::Down)
         {
             // Action Mode: Move focus to the previous item in the list.
-            SelectNextItem(true);
+            SelectNextItem(true, false);
+            e.Handled(true);
+        }
+        else if (key == VirtualKey::PageUp)
+        {
+            // Action Mode: Move focus to the previous item in the list.
+            SelectNextItem(false, true);
+            e.Handled(true);
+        }
+        else if (key == VirtualKey::PageDown)
+        {
+            // Action Mode: Move focus to the previous item in the list.
+            SelectNextItem(true, true);
```

Comment:
The original ticket also suggested support for Home and End. Consider implementing them as well :) In this case probably it makes sense to change the prototype of the SelectNextItem to receive enum.

Reply:
I was forgetting about a support for Home and End. Yeah enum would be do it  for this.